src/main/java/Duke.java

```diff
@@ -1,10 +1,180 @@
+                    System.out.println(todo);
+                    break;
+                case "D":
+                    Deadline deadline = new Deadline(description, time);
+                    System.out.println(deadline);
+                    break;
+                case "E":
+                    Event event = new Event(description, time);
+                    System.out.println(event);
+                    break;
+                default:
+                    System.out.println("       Unknown Object");
+                }
+            }
+        }
+        System.out.println("    ____________________________________________________________");
+    }
+
+    private static void addList() {
+        String line;
+        Scanner in = new Scanner(System.in);
+        // get user input
+        line = in.nextLine();
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
+                showList();
+                line = in.nextLine();
+                continue;
+            }
+            if (line.startsWith("done")) {
+                int index = Integer.parseInt(line.substring(line.length() - 1));
+                markAsDone(index);
+                line = in.nextLine();
+                continue;
+            }
+            System.out.println("    ____________________________________________________________");
+            System.out.println("     Got it. I've added this task: ");
+            done.add(0);
+            String description;
+            String time;
+            if (line.startsWith("todo")) {
+                type.add("T");
+                description = line.substring(5);
+                Todo todo = new Todo(description);
+                list.add(description);
+                date.add("");
+                System.out.println("       " + todo);
+            } else if (line.startsWith("deadline")) {
+                type.add("D");
+                int seperate = line.indexOf("/by");
+                description = line.substring(9, seperate - 1);
+                time = line.substring(seperate + 4);
+                Deadline deadline = new Deadline(description, time);
+                list.add(description);
+                date.add(time);
+                System.out.println("       " + deadline);
+            } else if (line.startsWith("event")) {
+                type.add("E");
+                int seperate = line.indexOf("/at");
+                description = line.substring(6, seperate - 1);
+                time = line.substring(seperate + 4);
+                Event event = new Event(description, time);
+                list.add(description);
+                date.add(time);
+                System.out.println("       " + event);
+            } else {
+                type.add("unknown");
+            }
+            System.out.println("     Now you have " + list.size() + " tasks in the list.");
+            System.out.println("    ____________________________________________________________");
```

Comment:
I think cause you used this line a few times, it is possible to declare it as a constant and just call it when you want to print it, it will make the code neater.